### ISSUE TITLE
`AssertStatement` doesn't have a position

### ISSUE THREAD

**Issue by tribals** [c2nes/javalang#73]
Here is simple example:

```java
// Foo.java
class Foo {
    void foo(int x) {
        assert x > 42;
    }
}
```

Let's parse it:

```python
from javalang.parse import parse
from javalang.tree import AssertStatement

with open('Foo.java') as f:
    tree = parse(f.read())

path, assert_stmt = next(tree.filter(AssertStatement))

print(repr(assert_stmt.position))
```

Expected output:

```
Position(line=4, column=9)
```

Actual output:

```
None
```

Is there any reason why an `AssertStatement` doesn't have a position?

**Comment by c2nes** [comment]
@tribals if you haven't already, could you test against master? https://github.com/c2nes/javalang/pull/70 added position information for a number of AST nodes, but hasn't been included in a release yet

**Comment by tribals** [comment]
@c2nes I've tested it against master. Yes, now `AssertStatement` does have a position.

**Comment by tribals** [comment]
Do you plan to release new version? If yes, how soon?

**Comment by c2nes** [comment]
I've published 0.13.0 to PyPI which includes #70

**Comment by tribals** [comment]
Thank you!
